.github/CODEOWNERS

```diff
@@ -41,7 +41,8 @@
 /embrace_mobile/                         @jpmunz @joage
 /eventstore/                             @Xorima
 /eversql/                                @tomershay team@eversql.com
-/exim/                                   @JeanFred
+/exim/                                   @
```

Comment:
```suggestion
/exim/                                   @JeanFred
```

Reply:
Note: You can ignore the `Integration f5-distributed-cloud has a `CODEOWNERS` entry, but the codeowner is not a username or team.` error. It will be fixed soon on our side.